### ISSUE TITLE
[Bug]: Background is too vibrant

### ISSUE THREAD

**Issue by mvahaste** [illixion/vscode-vibrancy-continued#46]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The backdrop is too vibrant, it looks like I have digital vibrance at 100% for VSCode only.

Here's a screenshot I took:

![Current](https://user-images.githubusercontent.com/48407846/205459418-f7e9d4d4-102a-419d-b439-c8bbb089aa6a.png)

### Expected Behavior

Backdrop should be less vibrant.

Here's a screenshot my friend took (same wallpaper):

![Expected](https://user-images.githubusercontent.com/48407846/205459449-29442fc7-39e0-43fe-b916-f5f1901a8527.png)

### Steps To Reproduce

1. Install vibrancy.

It's happened every time I've used this extension, even on a fresh windows install, but never to my friends. I've also tried disabling all other extensions, changing themes and looking through my NVIDIA control panel settings.

I have had this problem for a long time, even back when I used the original vibrancy extension. It's probably something on my end, but I'm just hoping that creating an issue will at least give me some new leads.

### Environment

```markdown
- OS: Windows 11 22H2
- VSCode: 1.73.1
- Extension: 1.1.12
```

**Comment by illixion** [comment]
Hey, I've noticed this too on my PC with an Nvidia GPU, you could try disabling GPU rendering for VSCode and seeing if that fixes this. Follow this instruction: https://stackoverflow.com/a/58762168

**Comment by mvahaste** [comment]
> Hey, I've noticed this too on my PC with an Nvidia GPU, you could try disabling GPU rendering for VSCode and seeing if that fixes this. Follow this instruction: [stackoverflow.com/a/58762168](https://stackoverflow.com/a/58762168)

I tried this and unfortunately it doesn't work. I should also probably mention that I have an RTX 2060 in my main PC and my laptop has Intel HD Graphics 520, I have this issue on both machines.

**Comment by PhoenixCODE2322** [comment]
> 

i am having the same issue, can you try get your friends vscode installation ( like vscode has a portable mode guide, just tell him to send you the required files) and run it in your environment , this will help isolate the issue.

**Comment by PhoenixCODE2322** [comment]
@mvahaste  i think i found it , first confirm from your friend is he using MicaForEveryone . 

I am using MicaForEveryone -- > Steps -->
1. Click on Add Rule ( left bottom) 
2. Add Process Rule
3. Enter "Code"
4. Leave everything to default except , go to bottom and enable "blur Behind"

**Comment by osadasami** [comment]
@PhoenixCODE2322 

could you share what vibrancy settings you use?
I use
Opacity: 0.1
Theme: defaul dark
Type: auto
and can't get blur effect like on your screenshot, only a little blur

![2022-12-13_09-10-52](https://user-images.githubusercontent.com/13532220/207224908-5b720b26-d004-4fe4-8242-9c48e456d61b.png)


In Mica For Everyone I use only blur Behind

**Comment by PhoenixCODE2322** [comment]
I have the same settings as yours --> 
![image](https://user-images.githubusercontent.com/75798544/207230105-ed43e62e-e177-414e-b5d2-c4fa29c8b8f8.png)
![image](https://user-images.githubusercontent.com/75798544/207230133-0f30638f-04b7-4c62-8192-31fcae6f3469.png)


Also i don't know but i installed this -- > dotnet-sdk-3.1.425-win-x64

**Comment by mvahaste** [comment]
> @mvahaste i think i found it , first confirm from your friend is he using MicaForEveryone .
> 
> I am using MicaForEveryone -- > Steps -->
> 
> 1. Click on Add Rule ( left bottom)
> 2. Add Process Rule
> 3. Enter "Code"
> 4. Leave everything to default except , go to bottom and enable "blur Behind"
>    ![image](https://user-images.githubusercontent.com/75798544/207061147-f6df239c-db61-4652-ac5b-2770fa5cf2f6.png)
>    ![image](https://user-images.githubusercontent.com/75798544/207061269-2ac45b71-1211-48b8-880b-26a367215c82.png)

Hi, my neither of my friends use MicaForEveryone, it just works out-of-the-box for them.

I can confirm that this workaround does help, thanks. It's still a _little_ off but definitely not too vibrant like it was before.

**Comment by ZeChArtiahSaher** [comment]
Mica sucks because upon launch or window snap blur gets killed

**Comment by MaksimKosyanchuk** [comment]
I have the same problem, I found 2 .cc files in the extension folder, and manually changed the contrast, but I don’t know where to compile these files, who knows how these C++ files work in the extension?

**Comment by sthephanno1** [comment]
same problem 
This is how blur is supposed to look like
![image](https://github.com/illixion/vscode-vibrancy-continued/assets/70006664/c20ce35e-f4fe-4a06-95ec-dbce19b95634)
This is how it looks
![image](https://github.com/illixion/vscode-vibrancy-continued/assets/70006664/cc252c06-5216-465e-b943-c7c30390f095)
i think when i first used win11 in august last year, NVidia cards had a problem rendering the acrylic and it looked exactly like the one from this extension, oversaturated and with not much blur

**Comment by MaksimKosyanchuk** [comment]
> Та же проблема Вот как должно выглядеть ![образ](https://user-images.githubusercontent.com/70006664/246584663-c20ce35e-f4fe-4a06-95ec-dbce19b95634.png) размытие Вот как это выглядит ![образ](https://user-images.githubusercontent.com/70006664/246584778-cc252c06-5216-465e-b943-c7c30390f095.png) Я думаю, когда я впервые использовал win11 в августе прошлого года, у карт NVidia были проблемы с рендерингом акрила, и он выглядел точно так же, как из этого расширения, перенасыщенный и с небольшим размытием
I don't have a nvidia video card, and the same problem, it's not related to the video card

**Comment by MaksimKosyanchuk** [comment]
> same problem This is how blur is supposed to look like ![image](https://user-images.githubusercontent.com/70006664/246584663-c20ce35e-f4fe-4a06-95ec-dbce19b95634.png) This is how it looks ![image](https://user-images.githubusercontent.com/70006664/246584778-cc252c06-5216-465e-b943-c7c30390f095.png) i think when i first used win11 in august last year, NVidia cards had a problem rendering the acrylic and it looked exactly like the one from this extension, oversaturated and with not much blur

I don't have a nvidia video card, and the same problem, it's not related to the video card

**Comment by parazite7** [comment]
Yeah it looks terrible, the problem is with the vibrancy theme. When on solarized dark vibrancy theme the colours are slightly better. We need a theme with a pure acrylic background like the terminal below.
![1](https://github.com/illixion/vscode-vibrancy-continued/assets/27277935/020e251e-c4ac-4300-9552-d1780f013437)

**Comment by rainbowflesh** [comment]
This issue is also a commonplace, I have made many attempts, and the results all point to, ah, It's a feature on Windows 11, unfixable if windows not expose more upper-level API.

Reference:
https://github.com/EYHN/vscode-vibrancy/issues/112
https://github.com/MicaForEveryone/MicaForEveryone/issues/359

**Comment by a0w89fwa0f** [comment]
Any updates on this? Why do some people have the issue but others don't?

**Comment by zvkiel** [comment]
same issue with a 3080 on W11. installing micaforeveryone and adding Code to a rule and enabling Enable blur behind fixed it for me

**Comment by illixion** [comment]
Vibrancy Continued v1.1.84 now always uses the modern DWM backdrop API for `acrylic` on Windows 11, as well as adding `mica`/`tabbed` variants that use the wallpaper color for vibrancy, both of which should hopefully look better. Windows 10 unfortunately stays on the old API. This should hopefully remove the need for MicaForEveryone, given it seems to use a similar method.
